CMakeLists.txt

```diff
@@ -87,6 +87,11 @@ if(PACKAGE_TGZ OR PACKAGE_ZIP OR PACKAGE_RPM OR PACKAGE_DEB)
   include(cmake/release.cmake)
 endif()
 
+set(SHARED_MODEL_DISABLE_COMPATIBILITY FALSE)
```

Comment:
`SHARED_MODEL_DISABLE_COMPATIBILITY` should be defined as option, like `TESTING`, otherwise it doesn't work.